packages/oui-datagrid/README.md

```diff
@@ -152,7 +152,7 @@ Clicked row action 1: <span ng-if="$ctrl.action1Row">{{$ctrl.action1Row.lastName
 | Attribute         | Type            | Binding | One-time binding | Values                    | Default             | Description                                       |
 | ----              | ----            | ----    | ----             | ----                      | ----                | ----                                              |
 | `page-size`       | number          | @?      |                  |                           | 25                  | maximum number of rows to show on each pages      |
-| `rows`            | string          | <?      | yes              |                           |                     | rows to show                                      |
+| `rows`            | array<object>   | <?      | yes              |                           |                     | rows to show                                      |
 | `rows-loader`     | function        | &?      | yes              |                           |                     | gets all rows (returns a promise with all rows)   |
```

Comment:
Adding a link here to your new section could be interesting to relate content in a user-friendly way